uint64 timestamp
uint64 integral_dt	# integration time
uint64 error_count
float32 x		# acceleration in the NED X board axis in m/s^2
float32 y		# acceleration in the NED Y board axis in m/s^2
float32 z		# acceleration in the NED Z board axis in m/s^2
float32 x_integral	# velocity in the NED X board axis in m/s over the integration time frame
float32 y_integral	# velocity in the NED Y board axis in m/s over the integration time frame
float32 z_integral	# velocity in the NED Z board axis in m/s over the integration time frame
float32 temperature	# temperature in degrees celsius
float32 range_m_s2	# range in m/s^2 (+- this value)
float32 scaling

int16 x_raw
int16 y_raw
int16 z_raw
int16 temperature_raw
